src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -245,18 +247,33 @@ private async void ChangeStates(ControlState state)
                 // Load the team project list, show progress animation
                 try
                 {
-                    ToggleLoading(true);
                     Dispatcher.Invoke(projects.Clear);
-                    var newProjectList = await UpdateTeamProjects().ConfigureAwait(true); // need to come back to original UI thread. 
-                    newProjectList.ForEach(p => projects.Add(p));
+
+                    // If the main window is always on top, then an error occurs where
+                    //  the login dialog is not a child of the main window, so we temporarily
+                    //  turn topmost off and turn it back on after logging in
+                    bool oldTopmost = false;
+                    Application.Current.Dispatcher.Invoke(() =>
+                    {
+                        oldTopmost = Application.Current.MainWindow.Topmost;
+                        Application.Current.MainWindow.Topmost = false;
+                    });
+
+                    ToggleLoading(true, () => Task.Run(async () =>
+                    {
+                        var newProjectList = await UpdateTeamProjects().ConfigureAwait(true); // need to come back to original UI thread. 
+                        Dispatcher.Invoke(() => newProjectList.ForEach(p => projects.Add(p)));
+                    }));
+
+                    Application.Current.Dispatcher.Invoke(() => Application.Current.MainWindow.Topmost = oldTopmost);
                     ToggleLoading(false);
                     Dispatcher.Invoke(() => serverTreeview.ItemsSource = projects);
                 }
 #pragma warning disable CA1031 // Do not catch general exception types
                 catch (Exception e)
                 {
                     e.ReportException();
-                    Dispatcher.Invoke(() => MessageDialog.Show("Error when retrieving team projects"));
+                    Dispatcher.Invoke(() => MessageDialog.Show("Error when retrieving team projects")); // have seen crash bc 'dispatcher is suspended', looking for repro
```

Comment:
unable to repro - saw a crash where this operation was invalid because the dispatcher was suspended. could use BeginInvoke but would like to find a repro again